kuma/users/admin.py

```diff
@@ -23,20 +22,29 @@ class UserAdmin(BaseUserAdmin):
     Extends the admin view of users to show date_joined field
     add a filter on the field too
     """
-    list_display = ('username', 'fullname', 'email', 'title', 'organization',
-                    'location', 'content_flagging_email', 'tags',
+    list_display = ('username', 'fullname', 'email',
+                    'bio', 'website', 'revisions',
                     'date_joined', 'is_staff', 'is_active')
-    list_editable = ('content_flagging_email', 'tags')
     list_filter = ('is_staff', 'is_superuser', 'is_active', 'date_joined')
     ordering = ('-date_joined',)
     search_fields = ('username', 'homepage', 'title', 'fullname',
                      'organization', 'location', 'bio', 'email', 'tags__name')
 
-    formfield_overrides = {
-        NamespacedTaggableManager: {
-            'widget': TagWidget(attrs={'size': 45})
-        }
-    }
+    def revisions(self, obj):
+        """HTML link to user's revisions with count"""
+        link = urlparams(reverse('dashboards.revisions'),
+                         user=obj.username)
+        count = obj.wiki_revisions().count()
```

Comment:
I just noticed that the `wiki_revisions` is slicing the query so you'll always get 5 here. You probably want to duplicated some of what that method is doing here to get the full count. Also, experiment with the query b/c this will get called once for every user in the table so it could cause some database read usage we should be careful about.